app/src/test/java/com/pavesid/androidacademy/ui/details/DetailsViewModelTest.kt

```diff
@@ -0,0 +1,80 @@
+package com.pavesid.androidacademy.ui.details
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.lifecycle.Observer
+import com.pavesid.androidacademy.MainCoroutineRule
+import com.pavesid.androidacademy.TestHelper
+import com.pavesid.androidacademy.data.details.DetailsWithCredits
+import com.pavesid.androidacademy.repositories.MoviesRepository
+import com.pavesid.androidacademy.utils.Resource
+import io.mockk.MockKAnnotations
+import io.mockk.coEvery
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.verifyOrder
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@ExperimentalCoroutinesApi
+class DetailsViewModelTest {
+
+    @get:Rule
+    val mainCoroutineRule = MainCoroutineRule()
+
+    @get:Rule
+    val instantTaskExecutorRule = InstantTaskExecutorRule()
+
+    private val coroutineDispatcher = TestCoroutineDispatcher()
+
+    @MockK
+    lateinit var repository: MoviesRepository
```

Comment:
try 
` = mockk< MoviesRepository >()` instead of annotation
so you do not need `lateinit`